.clang-format

```diff
@@ -2,11 +2,13 @@
 # Defaults for all languages.
 BasedOnStyle:  Google
 
-# Setting ColumnLimit to 0 so developer choices about where to break lines are maintained.
-# Developers are responsible for adhering to the 120 character maximum.
-ColumnLimit:     0
+ColumnLimit: 120
```

Comment:
The reason the 120 char limit is not enforced via clang-format is that it screws up deliberate line breaks (or at least used to), greatly diminishing readability. Similar to 'black' it took a 'make this line as long as possible by joining subsequent lines' approach.  Maybe it's better these days at leaving intentional line breaks alone.

Reply:
Thanks makes total sense. Thanks for the context

Reply:
> Maybe it's better these days at leaving intentional line breaks alone.

I've never followed a *hard limit* for line length (preferring soft limits where you *try* to fit a line length, but sometimes exceed it when suitable, using the visual editor guidelines in VS or VS Code) because sometimes making the line just a *little bit longer* actually improves the readability vs a hard limit. e.g. Say you're working in the Chrome codebase or TensorFlow codebase with a hard limit of 80 columns, but you have two lines that are *just a few characters* over, and it splits the logic making it hard to visually follow/scan 😭:

![image](https://user-images.githubusercontent.com/1809166/233753429-66d5e7d3-93ee-458f-be25-3f4927f8ba80.png)